apps/central-scan/frontend/src/screens/ballot_eject_screen.tsx

```diff
@@ -222,39 +214,134 @@ export function BallotEjectScreen({ isTestMode }: Props): JSX.Element | null {
           }
         }
       }
-    } else {
-      isUnreadableSheet = true;
     }
   }
 
-  const allowBallotDuplication =
-    !isInvalidTestModeSheet &&
-    !isInvalidBallotHashSheet &&
-    !isUnreadableSheet &&
-    !(isOvervotedSheet && precinctScanDisallowCastingOvervotes);
-
   const backInterpretation = reviewInfo.interpreted.back.interpretation;
   const isBackIntentionallyLeftBlank =
     backInterpretation.type === 'InterpretedHmpbPage' &&
     backInterpretation.markInfo.marks.length === 0;
   const isBlankSheet =
     isFrontBlank && (isBackBlank || isBackIntentionallyLeftBlank);
 
-  const adjudicationTitle = isInvalidTestModeSheet
-    ? isTestMode
-      ? 'Official Ballot'
-      : 'Test Ballot'
-    : isInvalidBallotHashSheet
-    ? 'Wrong Election'
-    : isUnreadableSheet
-    ? 'Unreadable'
-    : isOvervotedSheet
-    ? 'Overvote'
-    : isBlankSheet
-    ? 'Blank Ballot'
-    : isUndervotedSheet
-    ? 'Undervote'
-    : 'Unknown Issue';
+  if (isOvervotedSheet) {
+    for (const contestId of overvoteContestIds) {
+      highlightedContestIds.add(contestId);
+    }
+  } else if (isUndervotedSheet && !isBlankSheet) {
+    for (const contestId of undervoteContestIds) {
+      highlightedContestIds.add(contestId);
+    }
+  }
```

Comment:
Update logic to only highlight overvoted contests when overvoted, and to not highlight any contests when a fully blank ballot.